amt/services/storage.py

```diff
@@ -42,25 +42,54 @@ def init(storage_type: str = "file", **kwargs: Unpack[WriterFactoryArguments]) -
 class FileSystemStorageService(Storage):
     def __init__(self, location: Path = Path("./tests/data"), filename: str = "system_card.yaml") -> None:
         self.base_dir = location
-        if not filename.endswith(".yaml"):
-            raise ValueError(f"Filename {filename} must end with .yaml instead of .{filename.split('.')[-1]}")
         self.filename = filename
-        self.path = self.base_dir / self.filename
 
     def write(self, data: dict[str, Any]) -> None:
         if not Path(self.base_dir).exists():
             Path(self.base_dir).mkdir()
         with open(self.path, "w") as f:
             dump(data, f, default_flow_style=False, sort_keys=False)
 
-    def read(self) -> Any:  # noqa
-        # todo: this probably has to be moved to 'global scope'
-        yaml.add_constructor("!include", yaml_include.Constructor(base_dir=self.base_dir))
+    def read(self, autoload: bool = True) -> Any:  # noqa
+        yaml.add_constructor("!include", yaml_include.Constructor(base_dir=self.base_dir, autoload=autoload))
```

Comment:
I think yaml is a global scope include and I wonder if changing things, like add_constructor or remove things.. is done for "all".. (I would assume so).. this means for all yaml calls, it adds or remove constructors.. this could potentioally break things. Can we make this a local scope solution?